src/pybind/mgr/cephadm/migrations.py

```diff
@@ -442,6 +447,21 @@ def migrate_6_7(self) -> bool:
         # was set to true. Therefore we have nothing to migrate for those daemons
         return True
 
+    def migrate_7_8(self) -> bool:
+        def nfs_remove_old_membership_from_grace_table() -> None:
+            """
+            NFS has changed rados cluster node id from str to int in 8.1. This function will remove
```

Comment:
```suggestion
            NFS has changed rados cluster node id from str to int in Tentacle. This function will remove
```
we shouldn't reference downstream versions in the upstream code